depscan/lib/csaf.py

```diff
@@ -1884,7 +1884,8 @@ def add_vulnerabilities(data, results, direct_purls, reached_purls):
     for r in results:
         c = CsafOccurence(r)
         new_vuln = c.to_dict()
-        agg_score.add(severity_ref.get(c.severity))
+        if severity_ref.get(c.severity):
```

Comment:
@prabhu I would prefer the pr I opened as it will also remove this severity from the cvss v3 object and that is important as csaf 2.0 will fail validation if we have values other than critical, high, medium, low, or none in the vulnerability listing.